src/main/java/org/oresat/uniclogs/services/SatnogsTransferService.java

```diff
@@ -0,0 +1,122 @@
+package org.oresat.uniclogs.services;
```

Comment:
This file looks like a carbon copy of `UniclogsEnvironment`.

There may have been a misunderstanding of what the service file is for that needs more in-person explanation.

But the elements that `UniclogsEnvironment` is not what's needed in a service, if that's what your impression was. Not only that, but we're likely going to axe `UniclogsEnvironment` anyways as it turns out to be completely unnecessary.

Anyways, the long-short of it is that nearly everything that's in `SatnogsTmLink.doStart()` is what should be managed by this service instead.

`SatnogsTmLink` should have the sole job of just checking if it is able to reach db.satnogs.org/api and see if it is alive and reachable, which is what determines the link's state.

`SatnogsTransferService` will have the responsibility of using `SatnogsTmLink` to fetch a specific endpoint of the API and *then* do all of the TLM deduplication.